decidim/decidim
# Fix preserving bold text in the rich text editor when pasting content

#### :tophat: What? Why?
When trying to paste content to the rich text editor that contains bold text, the bold text is lost when the text is imported. This fixes the issue.

I have modified the pasting logic in Quill by overriding the Clipboard module with some of the newer functionality available in the latest code but not in the released versions. This seemed to fix the issue with some slight modifications I added due to the "linebreak" module that we have in Decidim.

As a sidenote, I'd like to remind about the editor discussion at #7585. Probably with the same effort as fixing all these bugs and issues we could've already migrated over to something much better, like the TipTap editor mentioned in the related discussion (which seems the most potential replacement to me right now).

#### :pushpin: Related Issues
- Fixes #8982
- Related to #7585

#### Testing
- Go to any rich text editor in the admin panel, e.g. pages component
- Write some text to the rich text input and make something bold
- Cut the text from the editor using CTRL+X
- Paste the same test to the editor using CTRL+V
- See that the bold text is now preserved properly
